### DISCUSSION TITLE
Get suffixe of URIRef object

### DISCUSSION THREAD

**Discussion by AlexisPister** [RDFLib/rdflib#1603] [Q&A]
Hello,

From an URIRef object, I would like to get the suffixe of the URI. I did not find such function in the documentation.

For example, if I have this object :
`uri = URIRef("http://www.w3.org/2000/01/rdf-schema#isDefinedBy")`

I would like to get this string :
`isDefinedBy`

Thanks

**Comment by tgbugs** [comment]
You're looking for Graph.qname or Graph.compute_qname.
